homework.js

```diff
@@ -0,0 +1,65 @@
+
+const summation = function(n) {
+    let sum = 0;
+    for(let i = 1 ; i <= n ; i++ ){
+        sum = sum + i; }
+        return sum;
+    }
+
+const summationEven = function(n) {
+    let sum = 0 ;
+    for (let i = 1 ; i <= n ; i++){
+       if (i%2 == 0){
+        sum = sum + i;
+       }
+    }
+    return sum;
+}
+
+const avg = function(n1,n2,n3,n4){    
+       sum = 0;
+       const avg = [n1,n2,n3,n4]; 
+      for(let i = 0 ; i < avg.length ; i++)
+       { sum = sum + avg[i]; }
+       return  sum/ avg.length; 
+    }
+
+
+const reverse = function(word){
+let rev = word.reverse();
```

Comment:
you can't use the function `reverse()` on strings as it only works with arrays.
The trick to convert the string to array using `split("")`, using the `reverse()` function and then convert it back to string using `join()`